README.md

```diff
@@ -30,10 +30,60 @@ Make intuitve pixel art in this simple drawing tool written in _vanilla javascri
 | color swap       | C                   |
 
 
+## Launching and Running
+Get the source code locally by navigating to your desired directory.
+Clone the repositry using the following command:
+```bash
+git clone https://github.com/Kully/pixel-paint.git
+```
+Open the pixel-paint folder in the newly cloned folder.
+Launch index.html locally. It should open a tab in your broswer.
+The program is now running locally! Go and create your own art!
+
+
 ## Contributing
 
 One of the joys of posting this project on Github is to get people excited and wanting to contribute to them. Feel free to create Pull Requests for any changes that you think the app could benifit from, or something that you would like to use.
 
 Have a look at some of the [Issues](https://github.com/Kully/pixel-paint/issues) in the repo. If you see something you want to add or comment on, by all means do so!
 
+How to Contribute:
+1. Fork the repository.
+2. Click on the "Clone" button and copy your unique link. It should look something like:
+    ```bash
+    https://github.com/<username>/pixel-paint
+    ```
+3. Clone the repo:
+    ``` bash
+    git clone https://github.com/<username>/pixel-paint
+    ```
+4. Navigate inside the cloned repo:
+    ```bash
+    cd pixel-paint
+    ```
+4. Create a new branch:
+    ```bash
+    git checkout -b <branch-name>
+    ```
+5. Make changes and add your new features/improvements you think benefits the app. Have fun and be creative!
+6. Add the newly edited files to git:
+    ```bash
+    git add .
+    ```
+    Note: This command adds all edited files to be commited. To commit specific files, see this [git documentation](https://git-scm.com/docs/git-add).
+7. Commit your changes:
+    ```bash
+    git commit -m "Detailed description of changes"
+    ```
+8. Push your code to your branch:
+    ```bash
+    git push origin <branch-name>
+    ```
+9. Create your Pull Request:
```

Comment:
Are all these steps necessary for Step 9? Again, this seems like basic git practice that people should know.

It also might be confusing if people's user experiences of Github are different. Mobile vs Web? If Github changes their GUI in the future? etc.